### ISSUE TITLE
Accented characters

### ISSUE THREAD

**Issue by TimothyGu** [uclaacm/Westwood_Sans#23]
Accented Latin characters exist in several languages, including ones widely used at UCLA (e.g., Spanish, Chinese [Pinyin](https://en.wikipedia.org/wiki/Pinyin), Vietnamese, French). A good starting point would be the Unicode [Latin-1 Supplement](https://www.unicode.org/charts/PDF/U0080.pdf) block which covers most Western European languages. After that would be various [Latin Extended](https://stackoverflow.com/a/14897846/1937836) blocks.

**Comment by Kimeiga** [comment]
I agree that Latin-1 is a good starter altho i am ethnically biased towards including certain characters of latin extended haha

I believe there's a prominent feature of Glyphs that allows one to draw the diacritics individually and then apply them to anchor points on the letterforms to automatically create lots of accented characters. Would be worth looking into albeit i would like a new person to try to take that up if possible haahah
